### PR TITLE
Solution #1

### PR DESCRIPTION
**Pull request by erikagayan**

### CODE REVIEW

**Review by Tarasidze — CHANGES_REQUESTED** [review]
You need to add migrations to your pull request. You may use: "git add PATH_TO _YOUR_FILE" command or change .gitignore file.
Everything else is ok!

**Review by LiudmylaKulinchenko — CHANGES_REQUESTED** [review]
tests didn't pass

File: main.py [hunk 1]
```diff
@@ -1,7 +1,57 @@
...
+        ("Will", "Smith"),
+        ("Jaden", "Smith"),
+        ("Scarlett", "Johansson"),
+    ]
+
+    for actor in actors:
+        Actor.objects.create(
+            first_name=actor[0],
+            last_name=actor[1]
+        )
```
Comment (Oleg-s-s) [inline main.py:32]: move variables to top and then apply some operations. That will make your code easier to understand

**Review by Oleg-s-s — CHANGES_REQUESTED** [review]
Taras mentioned that you need migrations on a pr. Thats true but holding db itself on git is a crucial mistake. remove db from your pr

**Review by Oleg-s-s — CHANGES_REQUESTED** [review]
You still left your db in pr, it should be removed

**Pull request closed without merging**